patroni/api.py

```diff
@@ -686,6 +691,9 @@ def get_postgresql_status(self, retry=False):
             if result['role'] == 'replica' and self.server.patroni.ha.is_standby_cluster():
                 result['role'] = postgresql.role
 
+            if result['role'] == 'replica' and self.server.patroni.ha.is_sync_standby():
+                result['sync_standby'] = 1
```

Comment:
Here we want to use the most up-to-date `cluster` object:
```suggestion
            if result['role'] == 'replica' and cluster.is_synchronous_mode()\
                    and cluster.sync and postgresql.name in cluster.sync.members:
                result['sync_standby'] = True
```